src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        class="input inputBig"
```

Comment:
```suggestion
        class="input"
```
inpuBig redundant. make all styles in input class, and change differences for small input in input-small class
![Screenshot from 2023-04-08 15-57-22](https://user-images.githubusercontent.com/33029864/230722526-abc2f403-afd0-4954-8fb5-8cd3351be4ac.png)